### ISSUE TITLE
Win7 Show Desktop - Icon height on vertical panel

### ISSUE THREAD

**Issue by sicherist** [Zren/plasma-applets#59]
Hello,

first of all I would like to thank you for creating the Win7 Show Desktop widget!

I use a vertical panel in KDE and the Win7 Show Desktop - Icon is way too high because it scales with the width of the vertical control panel. Would it be possible to reduce the icon height or even make it adjustable? 

![kde screenshot](https://cloud.githubusercontent.com/assets/6120865/22181815/889d1b0c-e094-11e6-87f0-ab310978fead.png)

**Comment by Zren** [comment]
You still using v2? It should be using a sane size on vertical panel (~8px high) since v3.

**Comment by sicherist** [comment]
I tried both v2 and v6. Both show the same result. However I have KDE 5.5.5 (kubuntu 16.04) and have not yet tried on version >= 5.7.

**Comment by Zren** [comment]
You need to restart plasmashell for a widget's update to take effect (eg: relog). v6 does not work on kde 5.5.5 though.

If you're interested, you could try editing `~/.local/share/plasma/plasmoids/org.kde.plasma.win7showdesktop/contents/ui/main.qml` then restarting plasmashell by running the following command: `killall plasmshell; kstart5 plasmashell`

You probably just need to change `Layout.maximumWidth: 3 // + 5 = 8` to `Layout.maximumHeight: 3 // + 5 = 8` but here's how I fixed it properly:

https://github.com/Zren/plasma-applets/commit/af771fd71fa050a5cdebaff5cfdd0a0a24477b91

**Comment by sicherist** [comment]
Thank you, it worked! I edited
`~/.local/share/plasma/plasmoids/org.kde.plasma.win7showdesktop/contents/ui/main.qml`
changed
`Layout.maximumWidth: 3 // + 5 = 8`
to
`Layout.maximumHeight: 3 // + 5 = 8`
and restarted plasma shell
`killall plasmashell; kstart5 plasmashell`


One problem however that I encountered was that KDE didn't seem to uninstall v6 properly and after I uninstalled v6 and reinstalled v2, I still had v6 - which doesn't work on KDE 5.5.5. 

What I did was I uninstalled it again, and manually deleted the directory `~/.local/share/plasma/plasmoids/org.kde.plasma.win7showdesktop` and reinstalled v2.
